### PR TITLE
Default pomodoro variables

### LINKED ISSUE #116: Feature Request: Variables for default pomodoro/breaks length
I love this feature in tct, but I tend to like longer pomodoros and breaks (50/15). It would be great if user can set up default lengths together with the rest of config, but still will be able to update them temporarily on the fly.

My intuition is that it's a small change, so I would write it myself, if anyone else finds it useful.

### PR DESCRIPTION
**Pull request by saintsebastian**
implements #116 

Simple addition of variables to `config` file, later they are passed as pomodoro options. I also added short description to README.md. Feel free to add to this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,13 @@ Other commands while in pomodoro mode:
 
 ```
 
+To change default pomodoro and break durations set following variables (these should be numbers):
```
Comment (maxcell) [inline README.md]: I think this is great! Might be good to just let people know if it is in seconds/minutes/hours when setting them.

**Comment by maxcell** [comment]
Also one thing you should add is the env variables to the `sample.env`!

File: care.js [hunk 2]
```diff
@@ -368,6 +368,10 @@ var pomodoroHandlers = {
       });
     }
   },
+
+  runningDuration: parseInt(config.runningDuration),
```
Comment (ink8bit) [inline care.js]: it's better to  specify a radix when using parseInt (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt)

File: care.js [hunk 3]
```diff
@@ -368,6 +368,10 @@ var pomodoroHandlers = {
...
+
+  breakDuration: parseInt(config.breakDuration),
```
Comment (ink8bit) [inline care.js]: same here (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt)

**Comment by saintsebastian** [comment]
@ink8bit @maxcell thanks for the review! I am not 100% happy with wording in `README.md` and `sample.env`, so feel free to change it if you feel it could be clearer. 
I am tempted to add some tests, but that would require changing too much. So I've tested manually on iTerm2 with oh-my-zsh.

**Comment by saintsebastian** [comment]
@notwaldorf any chance you might have time to take a look at this?

**Comment by notwaldorf** [comment]
Thanks so much! Will do a release soon!

**Pull request merged**